docs/Testing.md

```diff
@@ -63,10 +66,11 @@ The Backus-Naur Form (BNF) for `IMPORTER_FORCE_DYNAMIC` is as follows.
             <inputString ::= <inputIndex> `:` <dimString>
              <dimString> ::= <dimIndex> | <dimIndex> `,` <dimString>
             <inputIndex> ::= <index>
-              <dimIndex> ::= <index>
+              <dimIndex> ::= <index> | <index> ':' <symbol>
                  <index> ::= -1 | <number>
                 <number> ::= <digit> | <digit><number>
                  <digit> ::= 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
+                <symbol> ::= 'a', 'b', 'c', ...
```

Comment:
I would add `'z'` at the end to finish the list though it is still no perfect, e.g. 
`<symbol> ::= 'a', 'b', 'c', ..., 'z'`

Reply:
Fixed. Thanks.